src/commands/crashes/upload-symbols.ts

```diff
@@ -68,7 +81,7 @@ export default class UploadSymbols extends AppCommand {
     }
 
     let pathToZipToUpload: string;
-    if (typeof(zip) === "string") {
+    if (typeof (zip) === "string") {
```

Comment:
Not a biggie but please try and avoid whitespace changes if possible?

Reply:
yeah no problem! this was a remnant of doing an auto-format for this file. i've reverted this and the one below.